canton/BUILD.bazel

```diff
@@ -1274,6 +1274,30 @@ proto_jars(
     srcs = glob(["community/ledger-api/src/main/protobuf/com/daml/ledger/api/v*/**/*.proto"]),
     grpc = True,
     java_conversions = True,
+    maven_artifact_prefix = "ledger-api-internal",
```

Comment:
this is not getting released anymore, it is for internal usage with lapiv2 (lot of dependencies in ledger api implementation)

Reply:
If there are dependencies on the v2 in 2.x, I would expect somewhere the changes that latch onto this new published artifact and use it :thinking:

Reply:
I can't think of many usage: canton does not support it anymore (no x nodes). Canton internals use it, but outside should not.
Please note: this is not published, just a necessary compile dependency for the code drop.

Reply:
Oh, the `maven_artifact_prefix` tricked me

Reply:
yes unfortunately that is needed by our proto support (fails in pom generation), tried to do it without at first, then I realized we anyway declare all the publishable artifacts in artifacts.yaml